### PR TITLE
fix(pf3): when isRequiered, asterisk is added to label

### PR DESCRIPTION
**Pull request by rvsia**
When pf3 field gets isRequired then an asterisk is added to label

### CODE REVIEW

File: src/pf-3-form-fields/form-fields.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ const FinalFormField = ({
         <FormGroup validationState={ invalid ? 'error' : null }>
           { label &&
           <Col md={ 12 } componentClass="label" className="control-label">
-            { !hideLabel && __(label) }
+            { !hideLabel && (rest.isRequired ? __(label).concat(' *') : __(label)) }
```
Comment (Hyperkid123) [inline src/pf-3-form-fields/form-fields.js]: For pf3 This should look like this: https://www.patternfly.org/pattern-library/forms-and-controls/field-labeling/

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

**Comment by Hyperkid123** [comment]
@rvsia also there is a another component at `pf3-form-fields/multiple-choice-list`

**Comment by rvsia** [comment]
@Hyperkid123 the multiselect doesn't wotk. Did the styling changes broke it or is it intended to not work?

**Comment by Hyperkid123** [comment]
@rvsia i know i am doing PR for that right now

**Comment by rvsia** [comment]
OLD REPO -> CLOSED

**Pull request closed without merging**